New Organizations Chapter (#72)
uyuni-project/uyuni-docs: modules/administration/pages/organizations.adoc

> [[organizations]]

Author: not in an ID

You might like to have an organization for each physical location within your business, or for different business functions.

When you have created your organizations, you can create and assign users to your organizations.
You can then assign permissions on an organization level, which applies by default to every user assigned to the organization.

You can also configure authentication methods for your new organization, including PAM and single sign-on.
For more information about authentication, see xref:modules/administration/pages/auth-methods.adoc[].

[NOTE]
====
You must be logged in as a {productname} administrator to create and manage organizations.
====

.Procedure: Creating a New Organization
. In the {productname} {webui}, navigate to menu:Admin[Organizations], and click btn:[Create Organization].
. In the [guimenu]``Create Organization`` dialog, complete these fields:
* In the [guimenu]``Organization Name`` field, type a name for your new organization.
The name should be between 3 and 128 characters long.
* In the [guimenu]``Desired Login`` field, type the login name you want to use for the organization's administrator.

> This must be a new administrator account, you will not be able to use an existing administrator account to log in to the new organization, including the one you are currently logged in with.

Reviewer: ```suggestion
This must be a new administrator account, you will not be able to use an existing administrator account to sign in to the new organization, including the one you are currently signed in with.
```

* In the [guimenu]``Desired Password`` field, type a password for the new organization's administrator.
Confirm the password by typing it again in the [guimenu]``Confirm Password`` field.

> Password strength is indicated by the coloured bar beneath the password fields.

Author: far out, my Australian was showing yesterday, clearly!

> * In the [guimenu]``Email`` field, type an email for the new organization's administrator.

Author: I thought "email" on its own was acceptable these days ;)

> * In the [guimenu]``Email`` field, type an email for the new organization's administrator.

Reviewer: > I thought "email" on its own was acceptable these days ;)

You are probably right.  You have the teenager around you daily ;) I must listen more to my nephews.

* In the [guimenu]``First Name`` field, select a salutation, and type a given name for the new organization's administrator.
* In the [guimenu]``Last Name`` field, type a surname for the new organization's administrator.
. Click btn:[Create Organization].

== Manage Organizations

In the {productname} {webui}, navigate to menu:Admin[Organizations] to see a list of available organizations.
Click the name of an organization to manage it.

From the menu:Admin[Organizations] section, you can access tabs to manage users, trusts, configuration, and states for your organization.

[NOTE]
====
Organizations can only be managed by their administrators.

> To manage an organization, ensure you are logged in as the correct administrator for the organization you want to change.

Reviewer: ```suggestion
To manage an organization, ensure you are signed in as the correct administrator for the organization you want to change.
```

====

=== Organization Users

Navigate to the [guimenu]``Users`` tab to view the list of all users associated with the organization, and their role.
Clicking a username takes you to the [guimenu]``Users`` menu to add, change, or delete users.

=== Trusted Organizations

Navigate to the [guimenu]``Trusts`` tab to add or remove trusted organizations.

> Establishing trust between organizations allow them to share content more easily between them, and gives you the ability to migrate clients from one organization to another.

Reviewer: ```suggestion
Establishing trust between organizations allow them to share content between them, and gives you the ability to migrate clients from one organization to another.
```

=== Configure Organizations

Navigate to the [guimenu]``Configuration`` tab to manage the configuration of your organization.

> This includes the use of staged contents, setting up crash reporting, and the use of of SCAP files.

Reviewer: ```suggestion
This includes the use of staged contents, setting up crash reporting, and the use of SCAP files.
```

> For more information about content staging, see xref:administration:content-staging.adoc[].

Reviewer: If there are two or more items, we'd back make this an itemized list.

Author: I put it into prose instead, it's a pretty complicated concept.
